app/controlplane/configs/config.devel.yaml

```diff
@@ -5,7 +5,7 @@
 server:
   http:
     addr: 0.0.0.0:8000
-    timeout: 1s
+    timeout: 10s
```

Comment:
We need now a longer timeout since the HTTP endpoint handler now will check if the cas backend is valid in some cases, that takes some time. 

We have currently configured 10 seconds in the similar gRPC endpoints so I thought of using the same.